src/mono/wasm/Wasm.Build.Tests/ProjectProviderBase.cs

```diff
@@ -127,17 +134,26 @@ public IReadOnlyDictionary<string, DotNetFileName> FindAndAssertDotnetFiles(
                         string pattern = $"^{prefix}{s_dotnetVersionHashRegex}{extension}$";
                         var match = Regex.Match(actualFilename, pattern);
                         if (!match.Success)
+                        {
+                            msg.AppendLine($"   FP no match expectedFilename:{expectedFilename} actualFile:{actualFile}");
```

Comment:
Maybe it's just me, but I'm having problems to understand what are the messages telling me.

- FP no match ..
- FP actual ..
- No FP no match ..
- No FP actual ..